03-Cartographic-Modelling-with-Forest-Inventories.Rmd

```diff
@@ -0,0 +1,796 @@
+Old growth stands boast the highest rates of biodiversity and provide
+critical habitat for rare, threatened, or endangered species.
+Additionally, they contribute to soil fertility by supporting intricate
+soil structures formed by micro-organisms, fungi, and plants.
+
+It is generally assumed that the structural complexity desired in a
+forest develops over time. As such, a stand is designated as old growth
+once it has reached a certain age without experiencing major
+disturbance. The length of time required for this designation varies by
+Biogeoclimatic Ecosystem Classification
+[BEC](https://cfcg.forestry.ubc.ca/resources/cataloguing-in-situ-genetic-resources/about-bec-and-bgc-units/)
+zone.
+
+Old growth forests are also highly valued for timber production due to
+their superior qualities. (Canadian Institute of Forestry, 2022) These
+trees have had the opportunity to grow slowly over long periods,
+resulting in denser wood grain and stronger lumber. Furthermore, their
+tall, straight trunks are ideal for producing a wide range of wood
+products.
+
+To balance the benefits of both logging and preserving old growth
+forests, frameworks and guidelines are put in place to manage Old Growth
+sites. In BC, Old Growth Forests are assessed using the Cumulative
+Effects Framework (CEF). This framework was put in place to provide
+guidelines on assessing the effects of human activities and natural
+processes on the provinces forested ecosystems. The province’s old
+growth and mature-plus-old targets are outlined in the Biodiversity
+Guidebook (Province of B.C., 1995).
+
+Vancouver Island, British Columbia, has been an area of highly
+contentious Old Growth logging. Conflicting interests between logging
+companies, First Nations, Provincial and Federal Governments, and the
+general public came into the spotlight during the Fairy Creek protests
+of 2020. Over two years of protest, more than 1,000 protesters were
+arrested at the site (CBC, 2022). The events at Fairy Creek, and many
+others like it, have brought the issues of Old Growth conservation to
+the forefront of timber production relations.
+
+------------------------------------------------------------------------
+
+## Learning Objectives {.unnumbered}
+
+By the end of this lab, you will be able to:
+
+-   Prepare and clean a forest inventory dataset and related layers for
+    analysis
+
+-   Perform attribute/select-by-expression filtering to isolate
+    old-growth candidate polygons
+
+-   Calculate old-growth area and percentages by polygon using field
+    calculations
+
+-   Produce a cartographic summary (map + table) that communicates
+    old-growth distribution
+
+-   Reflect on limitations and assumptions of VRI-derived old-growth
+    estimates
+
+------------------------------------------------------------------------
+
+## Deliverables {.unnumbered}
+
+Submit the following:
+
+-   A map showing the study area and the distribution of old-growth
```

Comment:
Remove the list items below `-` and wrap each item with `<input type="checkbox" unchecked> ITEM HERE </input>` to produce the check boxes. Also, add the point values to each in brackets eg `Responses to the questions posed throughout the lab on the course management system (60 points)`.